docs/Troubleshooting/Installation.md

```diff
@@ -10,6 +10,11 @@ description: >
 The cause is the `devlake` container trying to decrypt data in the database with the wrong key.
 Please check the [GettingStarted/Upgrade](../GettingStarted/Upgrade.md) doc for more detail.
 
+### `process was terminated unexpectedly` error
+
+when collect datas, the task was failed.  The cause is the `devlake` container  and `grafana` container 's volumes not have the permissions to write.  it need to change the permissions of the volumes mounting directory of a and b to 777 
```

Comment:
I used GPT to paraphrase the sentence:
The data collection process encountered a failure due to insufficient permissions for writing in the volumes of the 'devlake' and 'grafana' containers. To resolve this, it is necessary to modify the permissions of the mounting directory for both 'a' and 'b' volumes to 777.

Reply:
yes

Reply:
@ningjunwei Then can you update the PR?  I can't modify it on your behalf.